cms/templates/js/content-group-editor.underscore

```diff
@@ -15,7 +15,7 @@
                     </span><%
                 }
                 %>
-                <input name="group-cohort-name" id="group-cohort-name-<%= uniqueId %>" class="collection-name-input input-text" value="<%- name %>" type="text" placeholder="<%= gettext("This is the name of the group") %>">
+                <input name="group-cohort-name" id="group-cohort-name-<%= uniqueId %>" class="collection-name-input input-text" value="<%= name %>" type="text" placeholder="<%= gettext("This is the name of the group") %>">
```

Comment:
An input value should not be escaped. With it escaped, the content group name is not shown correctly on edit, and on save it would be persisted as incorrectly escaped. This is how it looks on display:

![image](https://cloud.githubusercontent.com/assets/484484/13719728/2825b4de-e7c8-11e5-98f0-f8ea9d613944.png)

And then when editing (with incorrect escaping of value):

![image](https://cloud.githubusercontent.com/assets/484484/13719734/39cbc4bc-e7c8-11e5-9b59-4bfe41f5252c.png)